hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -4,7 +4,19 @@
 
 This tutorial explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or a USB hard disk. Note that this feature does not work with all USB mass storage devices.
 
-## Program USB host boot mode (Raspberry Pi 3A+ and 3B only)
+See the [documentation](./) for the boot sequence and alternative boot modes (Network, USB device, GPIO or SD boot).
+
+Note that "USB Mass Storage Boot" is different from "USB Device Boot Mode", the device boot mode allows a Raspberry Pi connected to a computer to boot using files on that computer.
+
+[Pi 2B v1.2 up to 3](./#raspberry-pi-2b-v12-3a-3b-cm3)
+
+[Pi 3B+ CM3+](#raspberry-pi-3b-cm3)
```

Comment:
To get anchor-links like this working on raspberrypi.org you need to add explicit anchors - see e.g. https://raw.githubusercontent.com/raspberrypi/documentation/master/configuration/raspi-config.md for how this is done.

Reply:
I've added these links; but I'm not quite sure if it works on the webpage; it did on GitHub (but was working on GitHub before also, I believe).

Reply:
Github adds automatic-anchor-links to Markdown headings, which is helpful in general but not in this particular case :wink: 
To work on the website, the text in the anchor-link needs to match the anchor-name, so to get `[Pi 3B+ CM3+](#raspberry-pi-3b-cm3)` to work you'd need to use `<a name="raspberry-pi-3b-cm3">`. (But there's no need for the anchors to precisely match the name of the heading, so `[Pi 3B+ CM3+](#pi_3plus)` and `<a name="pi_3plus">` would also work)

I hope that all makes sense! :smiley:

Reply:
I think it should work now, I was using the anchor names as GitHub is doing, but this edit should 'probably' also work?
Though I can't really test it unless if it's live as a webpage?
I have no clue how these .md pages are directly converted to HTML and hosted, I do think it's nice, but have no clue on how to reproduce this at home (w/o figuring it all out from scratch :p)